python/sglang/srt/server_args.py

```diff
@@ -590,7 +602,6 @@ def add_cli_args(parser: argparse.ArgumentParser):
         # Multi-node distributed serving
         parser.add_argument(
             "--dist-init-addr",
-            "--nccl-init-addr",  # For backward compatbility. This will be removed in the future.
```

Comment:
Why remove this?

Reply:
Deleting this parameter will require modifying many related commands in the documents.

Reply:
Added it back. I realized many documentation still use `--nccl-init-addr`. I have updated all of them to use `--dist-init-addr`. We will deprecate `--nccl-init-addr` next month.